setup.py

```diff
@@ -22,6 +22,6 @@
       url='https://github.com/mozilla/fxapom',
       license='MPL 2.0',
       packages=find_packages(exclude=['ez_setup', 'examples', 'tests']),
-      setup_requires=['setuptools_scm'],
+      setup_requires=['pytest-runner','setuptools_scm'],
```

Comment:
It's what I thought I needed to add, to get things working, from http://docs.pytest.org/en/latest/goodpractices.html#integrating-with-setuptools-python-setup-py-test-pytest-runner (which was linked to from http://tox.readthedocs.io/en/latest/example/basic.html#integration-with-setup-py-test-command).

I've since removed it, as tests now pass locally and in Travis (but not in Jenkins staging - still getting caught up on setup.py).